Sources/Mantis/CropViewConfig.swift

```diff
@@ -0,0 +1,20 @@
+import UIKit
+
+public struct CropViewConfig {
+    public var cropViewMinimumBoxSize: CGFloat = 42
+    public var cropViewPadding: CGFloat = 14
+    public var hotAreaUnit: CGFloat = 32
+    public var cropShapeType: CropShapeType = .rect
+    public var cropVisualEffectType: CropVisualEffectType = .blurDark
+    public var minimumZoomScale: CGFloat = 1
+    public var maximumZoomScale: CGFloat = 15
+    
+    /**
+     Rotation Dial currently is tightly coupled with other parts of CropView, we see rotation dial as a part of CropView,
+     so we put dialConfig inside CropViewConfig
+     */
+    public var dialConfig = DialConfig()
```

Comment:
Can't we make this thing optional and then we can get rid of `showRotationDial`?

Reply:
Fixed.